sample/src/main/java/com/google/accompanist/sample/swiperefresh/SwipeRefreshContentPaddingSample.kt

```diff
@@ -76,68 +80,59 @@ private fun Sample() {
         systemUiController.setSystemBarsColor(Color.Transparent, darkIcons = useDarkIcons)
     }
 
-    Surface {
-        Box(Modifier.fillMaxSize()) {
-            // A state instance which allows us to track the size of the top app bar
-            var topAppBarSize by remember { mutableStateOf(0) }
```

Comment:
The new implementation here avoids a recomposition loop caused by the `.onSizeChanged`